dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -362,6 +379,25 @@ class BGPPeerMgr(Manager):
         os.remove(tmp_filename)
         return rc == 0
 
+    def get_local_interface(self, local_addr):
```

Comment:
this method does not really belong to here. it belongs to InterfaceMgr

Reply:
Done it